src/js/features/arbitration/reducer.js

```diff
@@ -83,13 +83,26 @@ function reducer(state = DEFAULT_STATE, action) {
         }
       };
     case RESOLVE_DISPUTE_SUCCEEDED:
+    {
+      const arbitration = state.arbitration;
+      arbitration.arbitration.open = false;
+      arbitration.arbitration.result = action.result;
+
+      const currentEscrow = escrows.find(x => x.escrowId === action.escrowId);
+      if(currentEscrow) {
+        currentEscrow.arbitration.open = false;
+        currentEscrow.arbitration.result = action.result;
+      }
```

Comment:
This modifies the state directly and also, I just noticed that we have escrows inside the arbitration reducer. Does it make sense?
Do we duplicate the escrows in two reducers or in this one it's only about the arbitrations in escrows?

Reply:
It's only arbitrations in escrows!